service/dap/server_test.go

```diff
@@ -1269,12 +1269,31 @@ func TestPanicBreakpointOnNext(t *testing.T) {
 				execute: func() {
 					handleStop(t, client, 1, 5)
 
-					client.NextRequest(1)
-					client.ExpectNextResponse(t)
-
-					se := client.ExpectStoppedEvent(t)
-					if se.Body.ThreadId != 1 || se.Body.Reason != "panic" {
-						t.Errorf("\ngot  %#v\nwant ThreadId=1 Reason=\"panic\"", se)
+					// Send a next request until the panic breakpoint is hit.
+					// We expect to hit the unrecovered panic breakpoint and
+					// return a "panic" stopped event.
+					// In Go 1.13, 'next' will step into the defer in the runtime
```

Comment:
I would suggest being explicit in the test and having a separate block for Go 1.13 vs after. And maybe even skip 1.13 completely in your test. With the next Go version, 1.13 will probably be removed from the build set-up, but the complicated test will stay forever. I think you can use goversion.VersionAfterOrEqual to detect the version.

Reply:
Done!